src/Model/MenuSet.php

```diff
@@ -24,7 +30,11 @@ class MenuSet extends DataObject implements PermissionProvider
     ];
 
     private static $has_many = [
-        'MenuItems' => MenuItem::class,
+        'MenuItems' => Link::class,
+    ];
+
+    private static $owns = [
```

Comment:
[nitpick] since this is a new property

```suggestion
    private static array $owns = [
```